docs/product/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Product Walkthroughs
+title: What Is Sentry?
```

Comment:
```suggestion
title: What Is Sentry?
```
Thanks for taking a stab at this! While changing this to "What is Sentry" works really nicely as a navigational experience if you're starting on the homepage. I feel like "Product Walkthroughs" work better as the title on the sidenav if you're navigating from within the docs. Product Walkthroughs is probably one of our most important/highly trafficked sections of the product docs and I'd be worried about changing that to "what's sentry", which is a pretty narrow and potentially vague heading.